ucore/install-ucore-minimal.sh

```diff
@@ -28,15 +31,7 @@ rpm-ostree install /tmp/rpms/ublue-os-ucore-addons-*.rpm
 ## CONDITIONAL: install ZFS (and sanoid deps)
 if [[ "-zfs" == "${ZFS_TAG}" ]]; then
     rpm-ostree install /tmp/rpms/zfs/*.rpm \
-      lzop \
-      mbuffer \
-      mhash \
-      perl-Capture-Tiny \
-      perl-Config-IniFiles \
-      perl-Data-Dumper \
-      perl-Getopt-Long \
-      perl-Sys-Hostname \
-      pv
```

Comment:
I think the only dependency we should include with ZFS is `pv`, not `perl-Sys-Hostname`. This will match what is done here in `ucore-minimal:zfs` with what we do in the `fedora-coreos:zfs` image.

Reply:
This list was pulled from the documentation, so i removed what was mentioned as included in that.

Reply:
I understand. `perl-Sys-Hostname` is a dependency which the docs fail to reference, though happily, the `sanoid` RPM does indeed pull it in.

I include `pv` in `fedora-coreos:zfs`, and thus wish it for `ucore-minimal:zfs`, because it's incredibly  useful for `zfs send/recv` operations, even without sanoid.